src/bin/tikv-ctl.rs

```diff
@@ -1082,7 +1082,45 @@ fn main() {
                  .short("f").long("file")
                  .takes_value(true)
                  .help("meta file path"))
-        );
+        )
+        .subcommand(
+            SubCommand::with_name("compact-whole-cluster")
+                .about("compact whole cluster in a column family in a specified range")
+                .arg(
+                    Arg::with_name("pd")
+                        .short("p")
+                        .takes_value(true)
+                        .help("pd address"),
+                )
+                .arg(
+                    Arg::with_name("db")
+                        .short("d")
+                        .takes_value(true)
+                        .default_value("kv")
+                        .help("kv or raft"),
+                )
+                .arg(
+                    Arg::with_name("cf")
```

Comment:
Can we take multiple values for this argument?

Reply:
Done.